test/unit/math/rev/mat/prob/bernoulli_logit_glm_lpmf_test.cpp

```diff
@@ -37,7 +38,7 @@ TEST(ProbDistributionsBernoulliLogitGLM,
   for (size_t ii = 0; ii < 200; ii++) {
     Matrix<int, Dynamic, 1> y(3, 1);
     for (size_t i = 0; i < 3; i++) {
-      y[i] = Matrix<uint, 1, 1>::Random(1, 1)[0] % 2;
+      y[i] = abs(Matrix<int, 1, 1>::Random(1, 1)[0] % 2);
```

Comment:
Does this not work with `unsigned int` (instead of `uint`)? If it does, maybe we should test that instead?

Reply:
I just checked... it does work with `unsigned int` and I changed all the instances with that so it looks like the old code.